src/spearman.jl

```diff
@@ -43,37 +43,37 @@ immutable SpearmanCorrelationTest <: CorrelationTest
     # Spearman's ρ
     ρ::Float64
 
-    function SpearmanCorrelationTest(x,y)
+    function SpearmanCorrelationTest(x, y)
 
         n = length(x)
         (n != length(y)) && throw(ErrorException("x and y must have the same length"))
```

Comment:
Is there a reason this doesn't use the `@assert` macro? A la
`@assert n==length(y) "Variables x and y must have the same length"`